roles/postgresql/defaults/main.yml

```diff
@@ -89,6 +93,9 @@ postgresql_generated_users: >-
     {%- if groups['jupyterhub_server'] is defined and groups['jupyterhub_server'] | length > 0 -%}
       { "name": "{{ postgresql_jupyterhub_user }}", "password": "{{ postgresql_jupyterhub_password }}" },
     {%- endif -%}
+    {%- if groups['hue_server'] is defined and groups['hue_server'] | length > 0 -%}
+      { "name": "{{ postgresql_hue_user }}", "password": "{{ postgresql_hue_password }}", "auth_method": "md5" },
```

Comment:
Hue does not have a recent JDBC driver to use `scram-sha-256` ? Did you test without the `auth_method` ?

Reply:
md5 is proposed in Cloudera doc. By the way, the PostgreSQL version in Cloudera doc is 9.5. 
I did not see any problem with scram-sha-256 forw now.

Reply:
This is not related to PostgreSQL server version, this is related to the JDBC driver inside Hue if the driver is included. The JDBC driver is included during maven build or you give to Hue the driver like we do with the package `postgresql-jdbc` ?

Reply:
auth_method md5 is removed by last commit.